testing/resources/docker_hooks_repo/hooks.yaml

```diff
@@ -0,0 +1,11 @@
+-   id: docker-hook
+    name: Docker test hook
+    entry: echo
+    language: docker
+    files: \.txt$
+
+-   id: docker-hook-failing
+    name: Docker test hook with nonzero exit code
+    entry: bork
```

Comment:
actually nah, that's a bunch of complexity.  We can always change it later if people think otherwise.